predicthq/kafka/phq_kafka_processor.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from typing import List, Dict
+
+import confluent_kafka
+
+from .consumer import PhqKafkaConsumer, Message
+from .producer import PhqKafkaProducer
+
+log = logging.getLogger(__name__)
+
+
+class PhqKafkaProcessor(PhqKafkaConsumer):
```

Comment:
i dont see the need of this... for simplicity and readability of the service code that is using this lib, we can just provide consumer and producer

Reply:
originally there was only this one, (to keep it consistent with current services), I created the Consumer and Producer one latter on.
But I'm ok to remove that,